src/edgeProxy/edgeProxy.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+"use strict";
+
+const app = require("./app");
+const debug = require("debug")("edgeproxy:server");
+const http = require("http");
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+const port = normalizePort(process.env.PDS_EDGEPROXY_PORT || "4000");
+app.set("port", port);
+
+/**
+ * Create HTTP server.
+ */
+
+const server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
+server.on("error", onError);
+server.on("listening", onListening);
+
+/**
+ * Normalize a port into a number, string, or false.
+ */
+
+function normalizePort(val) {
+    const port = parseInt(val, 10);
+
+    if (isNaN(port)) {
+        // named pipe
+        return val;
+    }
+
+    if (port >= 0) {
+        // port number
+        return port;
+    }
+
+    return false;
+}
+
+/**
+ * Event listener for HTTP server "error" event.
+ */
+
+function onError(error) {
+    if (error.syscall !== "listen") {
+        throw error;
+    }
+
+    const bind = typeof port === "string"
+        ? "Pipe " + port
+        : "Port " + port;
+
+    // handle specific listen errors with friendly messages
+    switch (error.code) {
+    case "EACCES":
+        console.error(bind + " requires elevated privileges");
+        process.exit(1);
+        break;
+    case "EADDRINUSE":
+        console.error(bind + " is already in use");
+        process.exit(1);
+        break;
+    default:
+        throw error;
+    }
+}
+
+/**
+ * Event listener for HTTP server "listening" event.
+ */
+
+function onListening() {
+    const addr = server.address();
+    const bind = typeof addr === "string"
+        ? "pipe " + addr
+        : "port " + addr.port;
+    debug("Listening on " + bind);
```

Comment:
Given this is primarily an instructional artefact I think it would be helpful if the listening message was a bit more prominent - by default, debug messages go nowhere. Probably reasonable to log something to the console like 
````
"Personal data server edge proxy listening on port " + addr
````
and perhaps also print to the console the URL which can be used to access the main "Edge Proxy Example" view page

Reply:
Good point, @amb26. This has been addressed and ready for another look. Thanks.